OpenEdXMobile/src/test/java/org/edx/mobile/view/CourseOutlineActivityTest.java

```diff
@@ -1,291 +0,0 @@
-package org.edx.mobile.view;
-
-import android.app.Activity;
-import android.content.Intent;
-import android.os.Bundle;
-import android.support.v4.app.Fragment;
-import android.view.View;
-import android.widget.ListView;
-
-import org.edx.mobile.R;
-import org.edx.mobile.course.CourseAPI;
-import org.edx.mobile.http.provider.OkHttpClientProvider;
-import org.edx.mobile.model.api.EnrolledCoursesResponse;
-import org.edx.mobile.model.course.BlockPath;
-import org.edx.mobile.model.course.BlockType;
-import org.edx.mobile.model.course.CourseComponent;
-import org.edx.mobile.model.course.CourseStructureV1Model;
-import org.edx.mobile.model.course.IBlock;
-import org.junit.Test;
-import org.junit.runners.Parameterized.Parameters;
-import org.robolectric.Robolectric;
-import org.robolectric.Shadows;
-import org.robolectric.shadows.ShadowActivity;
-import org.robolectric.util.ActivityController;
-
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.EnumSet;
-import java.util.List;
-
-import static org.assertj.android.api.Assertions.assertThat;
-import static org.assertj.core.api.Java6Assertions.assertThat;
-import static org.edx.mobile.http.util.CallUtil.executeStrict;
-import static org.junit.Assert.*;
-
-public class CourseOutlineActivityTest extends CourseBaseActivityTest {
```

Comment:
We should preserve the tests in this class by moving them into some other class cuz they seem helpful and testing the right stuff.
p.s. if fixing/updating the tests seems like an out-of-scope or big effort, you can create a separate ticket for that.